docs/detections/view-osquery-results.asciidoc

```diff
@@ -0,0 +1,46 @@
+[[view-osquery-results]]
+=== Examine Osquery results
+Osquery provides you with relevant, timely data that you can use to better understand and monitor your environment. When you run queries, results are indexed and then displayed the Results table, which you can filter, sort, and interact with.
+
+[float]
+[[osquery-result-types]]
+=== Results table
+The Results table displays results from single queries and query packs.
+
+[float]
+[[review-single-osquery-results]]
+==== Single query results
+
+Results for single queries appear on the *Results* tab. When you run a query, the number of agents queried and query status temporarily display in a status bar above the results table. Agent responses can be `Sucessful`, `Not yet responded` (pending), and `Failed`.
+
+[role="screenshot"]
+image::images/single-query-results.png[width=80%][height=80%][Shows query results]
+
+[float]
+[[review-pack-osquery-results]]
+==== Query pack results
+
+Results for each query in the pack appear in the *Results* tab. Click the expand button (image:images/pack-expand-button-osquery.png[Click markdown icon,20,20]) at the far right of each query row to display query results. The number of agents that were queried and their responses are shown for each query. Agent responses are color-coded. Green is `Sucessful`, `Not yet responded` (pending) is gray, and `Failed` is red.
+
+[role="screenshot"]
+image::images/pack-query-results.png[width=80%][height=80%][Shows query results]
+
+[float]
+[[investigate-osquery-results]]
+=== Investigate query results
+
+From the results table, you can:
+
+* Click the *View in Discover* button (image:images/discover-button-osquery.png[Click the View in Discover button,20,20]) to explore the results in Discover.
+* Click the *View in Lens* button (image:images/lens-button-osquery.png[Click the View in Lens button,20,20]) to navigate to Lens, where you can use the drag-and-drop *Lens* editor to create visualizations.
+* Click the *Timeline* button (image:images/timeline-button-osquery.png[Click Timeline button,20,20]) to investigate a single query result in Timeline or *Add to timeline investigation* to investigate all results. This option is only available for single query results.
```

Comment:
Seems unnecessary & repetitive to call these out as buttons. This seems to work just as well:
```suggestion
* Click *View in Discover* (image:images/discover-button-osquery.png[Click the View in Discover button,20,20]) to explore the results in Discover.
* Click *View in Lens* (image:images/lens-button-osquery.png[Click the View in Lens button,20,20]) to navigate to Lens, where you can use the drag-and-drop *Lens* editor to create visualizations.
* Click *Timeline* (image:images/timeline-button-osquery.png[Click Timeline button,20,20]) to investigate a single query result in Timeline or *Add to timeline investigation* to investigate all results. This option is only available for single query results.
```